pypeit/reduce.py

```diff
@@ -140,6 +141,9 @@ def __init__(self, sciImg, spectrograph, par, caliBrate,
         #   WARNING -- It is best to unpack here then pass around self.slits
         #      Otherwise you have to keep in mind flexure, tweaking, etc.
 
+        # TODO: The spatial flexure is not copied to the PypeItImage object if
```

Comment:
Question for @jhennawi

Reply:
Shouldn't we be performing the spatial flexure cross-correlaton on the combined images rather than on the single images, that is the only thing that makes any sense to me.

Reply:
This is outside the scope of this PR.  Can one of you, @profxj or @jhennawi , decide if this is worthy of raising an issue?

Reply:
@profxj can you clarify your point here or we can discuss it at the next meeting. I think I'm not following, but I think we should be flexure x-correlating the combined image.